configs/config_ref_example.json

```diff
@@ -24,7 +24,28 @@
                       "hellos": {
                         "helloworld": {
                           "type": "helloworld",
-                          "hello": "greeting"
+                          "hello": "greeting",
+                          "secret_ref": [
+                            {
+                              "store_name": "local.file",
+                              "key": "db-user-pass:password",
+                              "sub_key": "db-user-pass:password",
+                              "inject_as": "redisPassword"
+                            }
+                          ],
+                          "component_ref": {
+                            "config_store_ref": "config_demo",
+                            "secret_store_ref": "local.file"
```

Comment:
How about naming them "config_store" and "secret_store"? The suffix "_ref" is redundent here since they are in "component_ref"

Reply:
@seeflood  resolved